dafny/lcf/.gitignore

```diff
@@ -1 +1,5 @@
+# Compiled C# code
 *.dll
+# Test outputs
+*.trace
+*.thm
```

Comment:
Please could we fix the missing newlines at the end of files? I think there's a few examples of it in this PR.

I think there's a vscode setting for this: https://stackoverflow.com/a/44704969

Reply:
Fixed